### PR TITLE
Show in progress rollouts on overview

### PR DESCRIPTION
**Pull request by spadgett**
Display a message when a rollout is in progress for a deployment config or deployment.

/assign @TheRealJon

### CODE REVIEW

**Comment by spadgett** [comment]
/hold

Need to sort RCs

**Comment by spadgett** [comment]
/hold

one more style issue

**Comment by spadgett** [comment]
secrets test flake, @jhadvig fyi

```
1) Interacting with the create secret forms Webhook secret : edits webhook secret
   NoSuchElementError: No element found using locator: by.buttonText("Generate")
```

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/overview.jsx [hunk 1]
```diff
@@ -33,6 +33,18 @@ import {
 // https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
 const EMPTY_GROUP_LABEL = 'other resources';
 
+const DEPLOYMENT_REVISION_ANNOTATION = 'deployment.kubernetes.io/revision';
```
Comment (TheRealJon) [inline frontend/public/components/overview.jsx]: nit: group constant declarations at the top

File: frontend/public/components/overview.jsx [hunk 2]
```diff
@@ -33,6 +33,18 @@ import {
...
+const getDeploymentRevision = obj => {
+  const revision = _.get(obj, ['metadata', 'annotations', DEPLOYMENT_REVISION_ANNOTATION]);
+  return revision && parseInt(revision, 10);
+};
+
+const DEPLOYMENT_CONFIG_LATEST_VERSION_ANNOTATION = 'openshift.io/deployment-config.latest-version';
```
Comment (TheRealJon) [inline frontend/public/components/overview.jsx]: nit: group constant declarations at the top

**Comment by spadgett** [comment]
Moved constant declarations

**Pull request merged**